build/docker/gaia-taskenv/build

```diff
@@ -1,4 +1,2 @@
 #! /bin/bash -vex
-
-make -C git
```

Comment:
We're now using tc-vcs to checkout gaia, just like the tester image does via mozharness.